asapdiscovery-genetics/asapdiscovery/genetics/seq_alignment.py

```diff
@@ -0,0 +1,306 @@
+import subprocess
+
+import numpy as np
+import pandas as pd
+from Bio import AlignIO, SeqIO
+
+# BioPython
+from Bio.Seq import Seq
+from Bio.SeqRecord import SeqRecord
+from bokeh.layouts import gridplot
+from bokeh.models import ColumnDataSource, Range1d
+from bokeh.models.glyphs import Rect, Text
+
+# Bokeh imports
+from bokeh.plotting import figure, output_file, save
+
+
+class Alignment:
+    def __init__(self, blast_match: pd.DataFrame, query: str, results_dir: str):
+        """An alignment object
+
+        Args:
+            blast_match (pd.DataFrame): DataFrame with BLAST results
+            query (str): Descriptor of query sequence
+            results_dir (str): Path for directory where results will be saved
+        """
+        self.dir_save = results_dir
```

Comment:
Have the names match, same goes for below. 

```suggestion
        self.dir_save = dir_save
```